rpc/documentation/api.md

```diff
@@ -1476,3 +1478,43 @@ The user input can be either a hexadecimal string or a mnemonic word list.
 - `InvalidArgument`: The input is invalid.
 
 **Stability:** Unstable
+
+#### `StartTicketPurchase`
+
+The `StartTicketPurchase` method enables ticket purchases for stake mining in the wallet.
```

Comment:
no, you can still purchase tickets without using this.  It needs to be documented as an *automatic* ticket purchaser that runs based on user-set parameters.